README.md

```diff
@@ -169,7 +169,7 @@ If you are building for NVIDIA's Jetson platforms (Jetson Nano, TX1, TX2, AGX Xa
 
 Common
 ```bash
-conda install numpy ninja pyyaml mkl mkl-include setuptools cmake cffi
+conda install numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests
```

Comment:
Added the other missing dependencies here. `future` in particular is annoying, because it's in `install_requires` and therefore `python setup.py develop` may try to install it with `easy_install` if it's missing.

It's unclear to me why `six` isn't in `install_requires` by the way, and if `requests` is still used.